packages/components/pagination/src/components/prev.ts

```diff
@@ -0,0 +1,23 @@
+import { buildProps } from '@element-plus/utils'
+import type { ExtractPropTypes } from 'vue'
+import type Prev from './prev.vue'
+
+export const paginationPrevProps = buildProps({
+  disabled: Boolean,
+  currentPage: {
+    type: Number,
+    default: 1,
+  },
+  prevText: {
+    type: String,
+    default: '',
```

Comment:
**Suggestion:** remove the default `''` since for `String` type this is already derived by Vue itself.